netplugin/plugin/netplugin.go

```diff
@@ -220,3 +196,43 @@ func (p *NetPlugin) GetEndpointStats() ([]byte, error) {
 func (p *NetPlugin) InspectState() ([]byte, error) {
 	return p.NetworkDriver.InspectState()
 }
+
+//GlobalFwdModeUpdate update the forwarding mode
+func (p *NetPlugin) GlobalFwdModeUpdate(cfg Config) {
+	var err error
+
+	if p.NetworkDriver != nil {
```

Comment:
we will delete the ovsSwitch instance and corresponding ofnetagent. The endpoint will be removed from ofnet.